SensusService/DataStores/DataStore.cs

```diff
@@ -249,7 +251,7 @@ public virtual void Start()
             }
         }
 
-        public Task AddAsync(Datum datum, CancellationToken cancellationToken)
+        public Task AddAsync(Datum datum, CancellationToken cancellationToken, bool runLocalCommitOnStore)
```

Comment:
I think this is Wes's stuff. I didn't realize some o f his changes had been mixed in with mine.